cmd/dusk/server.go

```diff
@@ -299,16 +315,36 @@ func Setup() *Server {
 
 // Close the chain and the connections created through the RPC bus.
 func (s *Server) Close() {
-	// TODO: disconnect peers
-	// _ = s.c.Close(cfg.Get().Database.Driver)
-	s.rpcBus.Close()
-	s.grpcServer.GracefulStop()
+	// Cancel all goroutines long-lived loops
+	s.cancel()
+
+	// Close graphql server.
+	if s.gqlServer != nil {
+		if err := s.gqlServer.Close(); err != nil {
+			log.WithError(err).Warn("failed to close gql server")
+		}
+	}
+
+	if s.grpcServer != nil {
+		s.grpcServer.GracefulStop()
+	}
```

Comment:
Does the golang process still need to serve any data over gRPC?

Reply:
It's going to be removed on merging testnet branch. For now we need gRPC for maintaining healthy test-harness. Test-harness pushes stake and transfer transactions via grpc.